hazelcast/src/test/java/com/hazelcast/client/map/ClientMapBasicTest.java

```diff
@@ -873,6 +874,26 @@ public void testRemoveAsync() throws Exception {
         assertNull(map.get(key));
     }
 
+    @Test
+    public void testReplaceAllWithStaticSerializableFunction() {
```

Comment:
Can you also add a test where the function throws an exception ? 
We would like to see that exception is delegated to the user correctly.